VideoLocker/res/layout/row_discussion_comments_comment.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="-5dp"
```

Comment:
The extra padding beyond the content area seems to be due to the elevation, so it should be equal to `@dimen/cardview_default_elevation` (2 DIP) on both sides (so it could be negated by settings the top margin to -4 DIP I guess). If you forgo the elevation by setting the `cardElevation` and `cardMaxElevation` to `0dp`, then the padding should be automatically reduced to the content area.